macros/GroupData.json

```diff
@@ -1932,14 +1932,31 @@
                             ]
         },
         "Web Crypto API": {
-            "overview":   [ "Web Crypto API" ],
-            "interfaces": [ "Crypto",
-                            "CryptoKey",
-                            "RandomSource",
-                            "SubtleCrypto" ],
-            "methods":    [],
-            "properties": [ "Window.crypto" ],
-            "events":     []
+            "overview":     [ "Web Crypto API" ],
+            "interfaces":   [ "Crypto",
+                              "CryptoKey",
+                              "CryptoKeyPair",
+                              "RandomSource",
+                              "SubtleCrypto" ],
+            "methods":      [],
+            "properties":   [ "Window.crypto" ],
+            "events":       [],
+            "dictionaries": [ "RsaPssParams",
```

Comment:
The dictionaries should be listed alphabetically.

Reply:
Done! Note though that in most cases in this file "dictionaries" is not sorted. We should lint for this kind of thing.